iModelCore/ecobjects/src/ECcontext.cpp

```diff
@@ -399,6 +399,12 @@ struct CustomAttributeInstanceReadContext: public ECInstanceReadContext
         if (it != references.end())
             return it->second.get();
 
+        if (m_schemaContext.GetSchemasToPrune().end() != std::find(m_schemaContext.GetSchemasToPrune().begin(), m_schemaContext.GetSchemasToPrune().end(), key.GetName()))
+            {
+            LOG.errorv("CustomAttributeInstanceReadContext - Custom attribute schema %s is one of the schemas that are to be pruned. Skipping fallback mechanism.", key.GetFullSchemaName().c_str());

```

Comment:
In this case, it's not an error, log as info or warning

Reply:
Changed to log it as the warning

Reply:
I still think finding hundreds of these warnings in their logs will confuse the connector people a lot.
Better make it clear that this is nothing that needs fixing, it is intended behavior. Somebody requested the schema to be pruned, so we are pruning a custom attribute which uses the schema.
Do not mention the fallback mechanism here. In the error message further down this is necessary because it is an exceptional case where we could not find a CA schema in the referenced schemas, so we are calling locateSchema as a fallback, ideally people would fix their schema by adding a reference.
But in this case with a pruned schema, fallback is not necessary or intended, and the schema is just fine.
I'd say just log an info saying "Skipping loading of a custom attribute because its schema (%s) is being pruned"